package.json

```diff
@@ -51,27 +49,6 @@
     "typedoc": "^0.24.0",
     "typescript": "^5.0.0"
   },
-  "@pika/pack": {
-    "pipeline": [
-      [
-        "@pika/plugin-ts-standard-pkg"
-      ],
-      [
-        "pika-plugin-merge-properties",
-        {
-          "properties": {
-            "octokit": "see https://github.com/jabuco/pika-plugin-merge-properties/issues/2"
-          }
-        }
-      ],
```

Comment:
@gr2m Do you recall why this was needed

Reply:
Not sure, but we can most likely ignore it.